### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by KyuuzoT**
08.06.2020 I've spotted that two of three project are not belonging to the homework #3 folder. Thus I had to make another commit in order to get this projects to their true place. **Please, don't fine me please=)**

### CODE REVIEW

File: HW_03/Task1/Calculator.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        }
+        public double ModuleDivision(double firstVar, double secondVar)
+        {
+            return firstVar%secondVar;
+        }
+        public double AreaOfCircle(double radius)
+        {
+            return Math.PI*radius*radius;
+        }
+    }
```
Comment (VadzimPapko) [inline HW_03/Task1/Calculator.cs:31]: we need an empty line between methods

File: HW_03/Task1/Program.cs [hunk 2]
```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Threading;
+
+namespace Task1
+{
+    class Program
+    {
+        private static bool bRepeateFlag = true;
```
Comment (VadzimPapko) [inline HW_03/Task1/Program.cs:8]: bRepeateFlag - strange name

File: HW_03/Task1/Program.cs [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        static void Main(string[] args)
+        {            
+            while (bRepeateFlag)
+            {
+                Menu();
+            }
+            Thread.Sleep(1000);
+        }
+
+        private static void Menu()
```
Comment (VadzimPapko) [inline HW_03/Task1/Program.cs:18]: great idea

File: HW_03/Task1/Program.cs [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+            Console.WriteLine("First parameter: ");
+            firstValue = InputValue();
+
+            Console.WriteLine("Second parameter: ");
+            secondValue = InputValue();
+        }
+
+        private static double InputValue()
+        {
+
```
Comment (VadzimPapko) [inline HW_03/Task1/Program.cs:93]: remove this empty line

File: HW_03/Task2/Calculator.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+            return firstVar / secondVar;
+        }
+        public static double ModuleDivision(double firstVar, double secondVar)
+        {
+            return firstVar % secondVar;
+        }
+        public static double AreaOfCircle(double radius)
+        {
+            return Math.PI * radius * radius;
+        }
```
Comment (VadzimPapko) [inline HW_03/Task2/Calculator.cs:30]: the same

File: HW_03/Task1/Program.cs (hunk 2 above)
Comment (KyuuzoT) [reply HW_03/Task1/Program.cs:8]: There is a reason for such name. I use this variable as a flag variable to determine whether the program should be repeated or could be terminated. 'b' in its name means 'bool'. Considering the overall design of this program I don't see a better name for this variable.

File: HW_03/Task1/Program.cs (hunk 4 above)
Comment (KyuuzoT) [reply HW_03/Task1/Program.cs:93]: I beg your pardon didn't notice this line before PR.

**Pull request merged**